utils/imageUtils.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * pathUtils.js
+ * A common set of utility functions related to uploading / importing
+ * files.
+ */
+const { exec } = require("child_process");
+const fs = require("fs");
+const { reject } = require("lodash");
+const path = require("path");
+
+module.exports = {
+   convert: async (imagePath, extension, quality, resize) => {
+      switch (extension) {
+         case "heic":
+         case "jpg":
+         case "png":
+         case "tiff":
+         case "webp":
+            break;
+         default:
+            return;
+      }
+      const args = ["convert"];
```

Comment:
Is this above code checking extension type ?

Reply:
Yes, It is.